python/python/lancedb/table.py

```diff
@@ -1488,12 +1489,19 @@ def search(
                 else:
                     raise e
 
+        use_tantivy = False
```

Comment:
How does this code work with s3/gcs? will it crash?

Reply:
for S3/GCS it would throw a exception, see `query.py: 754`